docs/samples/python/Presidio_with_Flair_as_a_3rd_party_recognizer

```diff
@@ -0,0 +1,146 @@
+# Integrating external Flair model with Presidio
+# This is a basic way to include Flair as a 3rd party recognizer for Presidio-Analyzer.
+
+from presidio_analyzer import AnalysisExplanation, EntityRecognizer, RecognizerResult, AnalyzerEngine
+from presidio_analyzer.nlp_engine import NlpArtifacts
+
+from flair.data import Sentence
+from flair.models import SequenceTagger
+
+from typing import List
+
+class FlairRecognizer(EntityRecognizer):
+    
+    
+    def __init__(
+        self,
+        supported_language: str = "en",
+        supported_entities: List[str] = [],
+        ner_strength: float = 0.85,
```

Comment:
Is this in use?

Reply:
It is not all in use, I eliminate the ner_strength value.